### PR TITLE
Feature/LKE 180

### PR DESCRIPTION
**Pull request by linkurious-infra**
Do not merge yet !

This is a demo, used on jenkins env for testing the new nodeJs pipeline and helper methods

### CODE REVIEW

**Comment by enys** [comment]
Reviewable, but not mergeable, as we need to discuss steps of the pipeline, and needs other releases preferably before this one. @davidrapin

File: package.json [hunk 1]
```diff
@@ -41,8 +41,11 @@
   "scripts": {
     "make": "npm run lint && npm run build && npm run test",
     "lint": "tslint --fix --config tslint.json 'src/**/*'",
+    "lint:ci": "tslint -t checkstyle -o reports/checkstyle.xml --config tslint.json 'src/**/*'",
     "test": "nyc mocha",
+    "test:unit": "nyc --reporter=text-summary --reporter=cobertura --report-dir=reports  mocha",
     "build": "tsc",
+    "tsc": "tsc",
```
Comment (enys) [inline package.json:48]: Perhaps this command should actually be called 'compile'

Comment (davidrapin) [reply package.json:48]: ok to rename to "compile" to clarify

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@Library('linkurious-shared')_
```
Comment (davidrapin) [inline Jenkinsfile:1]: what is the trailing underscore for?

Comment (enys) [reply Jenkinsfile:1]: Not a typo :) https://www.jenkins.io/doc/book/pipeline/shared-libraries/

Look for the section on Annotation (no anchor)

**Comment by davidrapin** [comment]
for reference: https://linkurious.atlassian.net/browse/LKE-180

**Pull request merged**